clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheActiveEntity.java

```diff
@@ -285,24 +310,24 @@ private EhcacheEntityResponse invokeLifeCycleOperation(ClientDescriptor clientDe
       case DESTROY_SERVER_STORE:
         destroyServerStore(clientDescriptor, (DestroyServerStore) message);
         break;
+      case CREATE_CLUSTERED_MAP:
+        createConcurrentMap(clientDescriptor, (CreateClusteredMap) message);
```

Comment:
Indeed, I was wrong on the number of messages. The `get` does not cause a message. But since there is nothing that indicates if the map is in use or not, I still think creating it implicitly would just be easier. Of course, the only reason I am thinking about allowing the _implicit_ here is that we are protected by the lifecycle of the clustered tier and the entity itself above it.